File: BTT4/BTT4 (BTT3 UPDATE)/BTT3/BlockFriendGUI.cpp
#include <iostream>
#include "BlockFriendGUI.h"
#include "Controller.h"


void BlockFriendGUI::RequestBlockFriendGUI(string userID)
{
	Controller Controller_GUI;
	cout << "Input UserName of friend, you want to block" << endl;
	string friendName;
	cin >> friendName;
	Controller_GUI.RequestBlockFriend(userID, friendName);
}

BlockFriendGUI::BlockFriendGUI()
{
}


BlockFriendGUI::~BlockFriendGUI()
{
}

File: BTT4_PY1/BTT4_PY1/BlockFriendGUI.py
from Controller import Controller
class BlockFriendGUI(object):
	def RequestToBlockFriend(userID):
		contr = Controller
		print("input friend's userName that you want to block: ")
		userNameFriend= input()
		print("userNameFriend: ",userNameFriend)
		contr.RequestToBlockFriend(userID, contr.TakeUserID(userNameFriend))

